v1.16-4.8/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG BUILD_DEPS=" \
       wget bzip2 zlib-dev git linux-headers \
       automake autoconf libtool build-base \
       ruby-dev libc6-compat geoip-dev \
-      snappy-dev gnupg \
+      snappy-dev gnupg bash \
```

Comment:
add bash to $BUILD_DEPS, rdkafka plugin needs bash and build tools to build. after build done , $BUILD_DEPS will be remove.